Applications/Tizen_CAPI/capi_func.c

```diff
@@ -46,9 +46,9 @@ bool file_exists(const char *filename) {
 
 #define NN_RETURN_STATUS()         \
```

Comment:
can rename to NN_DESTROY_MODEL_RETURN_STATUS.

Reply:
Done